### ISSUE TITLE
Display  first field (instead of username) in entry lists

### ISSUE THREAD

**Issue by nicoursi** [Kunzisoft/KeePassDX#1522]
**Is your feature request related to a problem? Please describe.**

Some templates do not have usernames but they still have relevant information to be shown straight away on the entry lists. A clear example is the "email address" field for the email template. Currently it is difficult to distinguish among the different accounts without opening the entries.

**Describe the solution you'd like**

I see two possible solutions:
1. Always show the first field even if it is not a usernane
2. Find a way to mark the field that has to be shown on the entry list

**Comment by J-Jamet** [comment]
I agree, this is a behavior that needs to be improved, the second solution is the most sustainable. Maybe to manage with long click and a marker.

**Comment by nicoursi** [comment]
Thank you for taking this into consideration. I wonder if the second solution  could keep the compatibility with other cross platform clients.

**Comment by J-Jamet** [comment]
Yes, that's the question I ask myself too. Maybe the best solution for compatibility is the first one, in which case it is necessarily linked to the sorting of custom fields. https://github.com/Kunzisoft/KeePassDX/issues/1230
